packages/react/src/date-picker/DatePicker.tsx

```diff
@@ -85,20 +88,35 @@ export type DatePickerProps = DatePickerInputProps &
 	DatePickerBaseProps;
 
 export const DatePicker = ({
-	value,
-	onBlur: onBlurProp,
-	onChange,
-	onInputChange: onInputChangeProp,
-	minDate,
-	maxDate,
+	allowedDateFormats: allowedDateFormatsProp = acceptedDateFormats,
+	dateFormat = 'dd/MM/yyyy',
 	initialMonth,
-	yearRange,
 	inputRef,
 	invalid = false,
+	maxDate,
 	maxWidth = 'md',
-	dateFormat = 'dd/MM/yyyy',
+	minDate,
+	onBlur: onBlurProp,
+	onChange,
+	onInputChange: onInputChangeProp,
+	value,
+	yearRange,
 	...props
 }: DatePickerProps) => {
+	const allowedDateFormats = useMemo(
```

Comment:
Why not maintain the prop's name on line 91 and give this a new name that describes the applied changes?

Reply:
Because I don't know what to call it 😅